pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -134,7 +134,18 @@ def mirrored_publish(version):
         version: The repository version to mirror-publish
     """
     with RpmPublication.create(version, pass_through=False) as publication:
-        for (result, relative_path) in metadata_files_for_mirroring[str(version.repository.pk)]:
+        has_repomd_signature = metadata_files_for_mirroring.get("repodata/repomd.xml.asc", None)
+        has_sqlite = any([".sqlite" in href for href in metadata_files_for_mirroring.values()])
+
+        publication.package_checksum_type = CHECKSUM_TYPES.UNKNOWN
+        publication.metadata_checksum_type = CHECKSUM_TYPES.UNKNOWN
```

Comment:
I dislike this, but I also have no idea how to properly set these values, especially for the edge cases.  They're not really relevant unless we're doing the publish ourselves.